src/components/input/test/SelectNext-test.js

```diff
@@ -21,20 +21,26 @@ describe('SelectNext', () => {
    *        Whether to renders SelectNext.Option children with callback notation.
    *        Used primarily to test and cover both branches.
    *        Defaults to true.
+   * @param {MultiSelect | Select | SelectNext} [options.Component] -
```

Comment:
I'm not completely sure this JSDoc is correct

Reply:
The JSDoc here is only for information and not typed checked, so this is fine as long as the meaning is clear.